application_service/app_service.cc

```diff
@@ -358,7 +358,8 @@ void app_service_loop(spawned_children* kid, int read_fd, int write_fd) {
   bool continue_loop = true;
 
 #ifdef DEBUG
-  printf("\napplication service loop: %d %d\n", read_fd, write_fd);
+  printf("[%d] Application Service loop: read_fd=%d write_fd=%d\n",
+         __LINE__, read_fd, write_fd);
```

Comment:
This, and other `.cc` files, have similar changes to elaborate diagnostic messages for improved troubleshooting.